doc/user/advanced_constraints.rst

```diff
@@ -344,6 +347,14 @@ For specifics of the mathematical formulation of the available group constraints
 
 .. seealso:: The :ref:`built-in national-scale example <examplemodels_nationalscale_settings>`'s ``scenarios.yaml`` shows two example uses of group constraints: limiting shared capacity with ``energy_cap_max`` and enforcing a minimum shared power generation with ``supply_share_min``.
 
+
+``demand_share_per_timestep_decision``
```

Comment:
Maybe give an example of why this might be useful (e.g. the share of fuel types in the heating sector)?